packages/jaeger-ui/src/components/TracePage/TracePageHeader/AltViewOptions.tsx

```diff
@@ -88,7 +89,7 @@ export default function AltViewOptions(props: Props) {
           <Link
             to={prefixUrl(`/api/traces/${traceID}?prettyPrint=true`)}
             rel="noopener noreferrer"
-            target="_blank"
+            target={getTargetBlankOrTop()}
```

Comment:
why did you not think EmptyOrBlank would work here?

Reply:
The problem is that when the target is empty, the json will be shown in the same window and the user cannot return to the page before (at least on the instrument because it has no keyboard or possibility to reach a return button from the browser). This is why I decided to use top: when the target is top the user can still access the navigation bar and go back.